vms/platformvm/txs/txstest/builder.go

```diff
@@ -414,8 +430,18 @@ func (b *Builder) feeCalculator() *fees.Calculator {
 	if !isEActive {
 		feeCalculator = fees.NewStaticCalculator(b.cfg, chainTime)
 	} else {
+		nextChainTime, _, err := state.NextBlockTime(b.state, b.clk)
```

Comment:
when building a tx, we need to update the feeManager to use the right fee rates, i.e. the fee rates at the next block time